cmake/onnxruntime_java.cmake

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2019, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
That file was modified in 2022, I don't think I've made changes to it since then. Oracle's preferred copyright statement at the moment is `Copyright (c) <year-created>, <year-last-modified>, Oracle and/or its affiliates. All rights reserved.` so I've been updating them in that form.